decisions/components-preview.md

```diff
@@ -0,0 +1,14 @@
+# Component Preview (a.k.a stories)
+
+## Decision
+- All previews should respond to a shortcut to hide the navbar (CMD+D).
+- The component being previewed should fit in the screen in portrait mode *without* the navbar.
+- The IOS preview should be as close as possible as the Android version (Always respecting any platform specific behaviour).
+- Diferent configurations (e.g. normal text and emphasized text) should have different previews. In this case clicking the main story should show a new list view with all configs.
+- When converting a component from react native the previews should be as close as possible to the react native version.
```

Comment:
I would possibly remove this line - we made the mistake of making our RN stuff too "webby". If anything we should align the existing RN stuff to the new Android / iOS stuff.

IMO

Reply:
I see what you mean but this is more about the previews than the actual component. If the actual component is different then by all means the preview should be different to. Maybe I should clarify that

Reply:
👍

Reply:
Yeah, doesn't hurt to make it more explicit 👍